command/plugin_deregister_test.go

```diff
@@ -206,7 +206,7 @@ func TestPluginDeregisterCommand_Run(t *testing.T) {
 			t.Errorf("expected %d to be %d", code, exp)
 		}
 
-		expected := "Success! Deregistered plugin (if it was registered): "
+		expected := "does not exist in the catalog"
```

Comment:
nit: I wonder if we should change the test below to be a bit more specific by using `strings.HasSuffix` rather than `strings.Contains`.